handler.js

```diff
@@ -136,98 +132,101 @@ module.exports.start_build = (event, context, callback) => {
+                octokit.repos.createStatus(status).then(function (data) {
                   // success
-                  console.log("Set running status:", data)
-                  callback(null, response);
-                }).catch(function(err) {
-                  console.log(err);
-                  callback(err);
-                });
+                  console.log('Set running status:', data)
+                  callback(null, response)
+                }).catch(function (err) {
+                  console.log(err)
+                  callback(err)
+                })
               }
-            });
-          }).catch(function(err) {
-            console.log("Github authentication failed");
-            console.log(err, err.stack);
-            callback(err);
-          });
+            })
+          }).catch(function (err) {
+            console.log('Github authentication failed')
+            console.log(err, err.stack)
+            callback(err)
+          })
         }
-      });
+      })
     }
-  });
+  })
 }
 
 module.exports.check_build_status = (event, context, callback) => {
-  var response = event;
+  var response = event
   var params = {
     ids: [event.build.id]
   }
-  codebuild.batchGetBuilds(params, function(err, data) {
+
+  codebuild.batchGetBuilds(params, function (err, data) {
     if (err) {
-      console.log(err, err.stack);
+      console.log(err, err.stack)
       context.fail(err)
-      callback(err);
+      callback(err)
     } else {
       response.build = data.builds[0]
-      callback(null, response);
+      callback(null, response)
     }
-  });
+  })
 }
 
 module.exports.build_done = (event, context, callback) => {
   // get the necessary variables for the github call
-  var base = event.pull_request.base;
-  var head = event.pull_request.head;
-  var repo = base.repo;
+  var base = event.pull_request.base
+  var head = event.pull_request.head
+  var repo = base.repo
 
-  console.log('Found commit identifier: ' + head.sha);
+  console.log('Found commit identifier: ' + head.sha)
 
   // map the codebuild status to github state
-  var buildStatus = event.build.buildStatus;
-  var state = '';
-  switch(buildStatus) {
+  var buildStatus = event.build.buildStatus
+  var state = ''
+  switch (buildStatus) {
     case 'SUCCEEDED':
-      state = 'success';
-      break;
+      state = 'success'
+      break
     case 'FAILED':
-      state = 'failure';
-      break;
+      state = 'failure'
+      break
     case 'FAULT':
     case 'STOPPED':
     case 'TIMED_OUT':
       state = 'error'
+      break
     default:
       state = 'pending'
   }
-  console.log('Github state will be', state);
 
-  setGithubAuth(octokit, ssm, ssmParams, function (err) {
+  console.log('Github state will be', state)
+
+  setGithubAuth(ssm, ssmParams, function (err, octokit) {
     if (err) {
-      console.log(err);
-      callback(err);
+      console.log(err)
+      callback(err)
```

Comment:
i'm not super sure, but does the callback need to include two values (now that `octokit` is in the callback)
```suggestion
      callback(err, null)
```

Reply:
Oh, sorry, in the wrong function for that. derp.

Reply:
Given the other code that has multiple returns in their callback, I do not think so.